app/controllers/repos_controller.rb

```diff
@@ -80,6 +99,7 @@ def show_releases
     if (@form_result_type = params[:result_type]) &&
        (@benchmark_type = find_benchmark_type_by_category(@form_result_type))
 
+      versions_calculate_once = ActiveSupport::OrderedHash.new
```

Comment:
`versions_calculate_once` should probably be `versions_cache` or something more descriptive. It doesn't actually calculate once